### PR TITLE
Added Spanish translation by merlox (Merunas Grincalaitis) for the Bounty "Translate App Strings to Spanish #2332"

### PR DESCRIPTION
**Pull request by merlox**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

addresses #...

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
...

### Steps to test:
- Open Status
- ...
- Step 3, etc.

[comment]: # (PRs will only be accepted if squashed into single commit.)

status: ready

*or*

status: wip

### CODE REVIEW

**Comment by andytudhope** [comment]
Hey @merlox! Could you put the issue number into the actual PR body (i.e. the comment) as that is where the web hook is looking. We will review and get back to you, thanks for being the first ever Open Bounty submission 😉 🎉

**Comment by andytudhope** [comment]
Hey @merlox please close this PR and resubmit it and make sure to reference the issue number where the script says `Fixes #2332`. That way we can pick it up correctly.

**Review by AlessandroSpallina — CHANGES_REQUESTED** [review]
Hey, change first line from
(ns status-im.translations.en)
to
(ns status-im.translations.es)

File: src/status_im/translations/es.cljs [hunk 1]
```diff
@@ -1,51 +1,68 @@
-(ns status-im.translations.es)
+(ns status-im.translations.en)
```
Comment (andytudhope) [inline src/status_im/translations/es.cljs:1]: change to `(ns status-im.translations.es)`

**Pull request closed without merging**